### ISSUE TITLE
The get_package script doesn't actually have an "all" option even though the doc says so

### ISSUE THREAD

**Issue by utke1** [bradbell/cmpad#2]
https://github.com/bradbell/cmpad/blob/8e4c0502c6b33d3fefe92df417fd88e1c0c424d9/bin/get_package.sh#L37

when I try to run 
```
cpp/bin/get_package.sh release all
```
I get:
```
package = 'all' is not available
```

**Comment by utke1** [comment]
this is probably solved by addressing #3

**Comment by bradbell** [comment]
Do you have a suggestion for how the docs should be changed ?
